definitions/npm/express_v4.x.x/flow_v0.32.x-/test_response.js

```diff
@@ -14,6 +14,7 @@ app.use('/response_api', (req: express$Request, res: express$Response) => {
     'Content-Type': 'application/json',
   });
   res.end('{}');
+  console.log('request: ', res.req);
```

Comment:
I updated this test to match other [similar tests](https://github.com/nowait/flow-typed/blob/master/definitions/npm/express_v4.x.x/flow_v0.32.x-/test_overrideUseMethodClassExtension.js#L55)